[2023-03-08T17:24:06.689+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2023-03-08T17:24:06.694+0000] {local_task_job.py:147} INFO - Task is not able to be run
[2023-03-08T17:30:55.072+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [queued]>
[2023-03-08T17:30:55.101+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [queued]>
[2023-03-08T17:30:55.103+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T17:30:55.107+0000] {taskinstance.py:1280} INFO - Starting attempt 5 of 5
[2023-03-08T17:30:55.110+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T17:30:55.196+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonVirtualenvOperator): chatgpt_prediction> on 2023-03-08 17:00:28.852949+00:00
[2023-03-08T17:30:55.220+0000] {standard_task_runner.py:55} INFO - Started process 2009 to run task
[2023-03-08T17:30:55.258+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bitcoin_second_webscrape', 'chatgpt_prediction', 'manual__2023-03-08T17:00:28.852949+00:00', '--job-id', '218', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpffopgni9']
[2023-03-08T17:30:55.262+0000] {standard_task_runner.py:83} INFO - Job 218: Subtask chatgpt_prediction
[2023-03-08T17:30:55.488+0000] {task_command.py:388} INFO - Running <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]> on host cd114ba8c1f8
[2023-03-08T17:30:55.666+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bitcoin_second_webscrape
AIRFLOW_CTX_TASK_ID=chatgpt_prediction
AIRFLOW_CTX_EXECUTION_DATE=2023-03-08T17:00:28.852949+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-08T17:00:28.852949+00:00
[2023-03-08T17:30:55.674+0000] {process_utils.py:179} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvsm2wgtk3 --system-site-packages
[2023-03-08T17:30:55.698+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:30:57.275+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-03-08T17:30:57.278+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-03-08T17:30:57.285+0000] {local_task_job.py:147} INFO - Task is not able to be run
[2023-03-08T17:30:57.470+0000] {process_utils.py:187} INFO - created virtual environment CPython3.7.16.final.0-64 in 1089ms
[2023-03-08T17:30:57.474+0000] {process_utils.py:187} INFO -   creator CPython3Posix(dest=/tmp/venvsm2wgtk3, clear=False, no_vcs_ignore=False, global=True)
[2023-03-08T17:30:57.476+0000] {process_utils.py:187} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-03-08T17:30:57.479+0000] {process_utils.py:187} INFO -     added seed packages: pip==23.0.1, setuptools==67.2.0, wheel==0.38.4
[2023-03-08T17:30:57.483+0000] {process_utils.py:187} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-03-08T17:30:57.515+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvsm2wgtk3/bin/pip install -r /tmp/venvsm2wgtk3/requirements.txt
[2023-03-08T17:30:57.541+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:31:00.466+0000] {process_utils.py:187} INFO - Collecting selenium
[2023-03-08T17:31:00.560+0000] {process_utils.py:187} INFO -   Using cached selenium-4.8.2-py3-none-any.whl (6.9 MB)
[2023-03-08T17:31:00.622+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 2)) (1.3.5)
[2023-03-08T17:31:00.706+0000] {process_utils.py:187} INFO - Collecting gcsfs
[2023-03-08T17:31:00.712+0000] {process_utils.py:187} INFO -   Using cached gcsfs-2023.1.0-py2.py3-none-any.whl (26 kB)
[2023-03-08T17:31:00.717+0000] {process_utils.py:187} INFO - Requirement already satisfied: fsspec in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 4)) (2022.11.0)
[2023-03-08T17:31:00.720+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-storage in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 5)) (2.7.0)
[2023-03-08T17:31:00.723+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 6)) (2.34.4)
[2023-03-08T17:31:00.725+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 7)) (2.16.0)
[2023-03-08T17:31:00.726+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-google in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (8.8.0)
[2023-03-08T17:31:00.728+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***[google] in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.5.1)
[2023-03-08T17:31:00.794+0000] {process_utils.py:187} INFO - Collecting openai
[2023-03-08T17:31:00.798+0000] {process_utils.py:187} INFO -   Using cached openai-0.27.0-py3-none-any.whl (70 kB)
[2023-03-08T17:31:00.810+0000] {process_utils.py:187} INFO - Requirement already satisfied: certifi>=2021.10.8 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venvsm2wgtk3/requirements.txt (line 1)) (2022.12.7)
[2023-03-08T17:31:00.869+0000] {process_utils.py:187} INFO - Collecting trio~=0.17
[2023-03-08T17:31:00.879+0000] {process_utils.py:187} INFO -   Using cached trio-0.22.0-py3-none-any.whl (384 kB)
[2023-03-08T17:31:00.889+0000] {process_utils.py:187} INFO - Requirement already satisfied: urllib3[socks]~=1.26 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venvsm2wgtk3/requirements.txt (line 1)) (1.26.14)
[2023-03-08T17:31:00.938+0000] {process_utils.py:187} INFO - Collecting trio-websocket~=0.9
[2023-03-08T17:31:00.944+0000] {process_utils.py:187} INFO -   Using cached trio_websocket-0.9.2-py3-none-any.whl (16 kB)
[2023-03-08T17:31:00.974+0000] {process_utils.py:187} INFO - Requirement already satisfied: numpy>=1.17.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvsm2wgtk3/requirements.txt (line 2)) (1.21.1)
[2023-03-08T17:31:00.976+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytz>=2017.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvsm2wgtk3/requirements.txt (line 2)) (2022.7.1)
[2023-03-08T17:31:00.978+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-dateutil>=2.7.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvsm2wgtk3/requirements.txt (line 2)) (2.8.2)
[2023-03-08T17:31:00.990+0000] {process_utils.py:187} INFO - Requirement already satisfied: decorator>4.1.2 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (5.1.1)
[2023-03-08T17:31:00.993+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-oauthlib in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (0.8.0)
[2023-03-08T17:31:01.152+0000] {process_utils.py:187} INFO - Collecting fsspec
[2023-03-08T17:31:01.158+0000] {process_utils.py:187} INFO -   Using cached fsspec-2023.1.0-py3-none-any.whl (143 kB)
[2023-03-08T17:31:01.163+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (2.28.2)
[2023-03-08T17:31:01.166+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiohttp!=4.0.0a0,!=4.0.0a1 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (3.8.3)
[2023-03-08T17:31:01.232+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-core<3.0dev,>=2.3.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvsm2wgtk3/requirements.txt (line 5)) (2.3.2)
[2023-03-08T17:31:01.234+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-resumable-media>=2.3.2 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvsm2wgtk3/requirements.txt (line 5)) (2.4.0)
[2023-03-08T17:31:01.236+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvsm2wgtk3/requirements.txt (line 5)) (2.8.2)
[2023-03-08T17:31:01.303+0000] {process_utils.py:187} INFO - Requirement already satisfied: packaging<22.0dev,>=14.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvsm2wgtk3/requirements.txt (line 6)) (21.3)
[2023-03-08T17:31:01.306+0000] {process_utils.py:187} INFO - Requirement already satisfied: protobuf<4.0.0dev,>=3.12.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvsm2wgtk3/requirements.txt (line 6)) (3.20.0)
[2023-03-08T17:31:01.308+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio<2.0dev,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvsm2wgtk3/requirements.txt (line 6)) (1.51.1)
[2023-03-08T17:31:01.310+0000] {process_utils.py:187} INFO - Requirement already satisfied: proto-plus<2.0.0dev,>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvsm2wgtk3/requirements.txt (line 6)) (1.19.6)
[2023-03-08T17:31:01.383+0000] {process_utils.py:187} INFO - Requirement already satisfied: six>=1.9.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvsm2wgtk3/requirements.txt (line 7)) (1.16.0)
[2023-03-08T17:31:01.386+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvsm2wgtk3/requirements.txt (line 7)) (4.2.2)
[2023-03-08T17:31:01.389+0000] {process_utils.py:187} INFO - Requirement already satisfied: rsa<5,>=3.1.4 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvsm2wgtk3/requirements.txt (line 7)) (4.9)
[2023-03-08T17:31:01.391+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvsm2wgtk3/requirements.txt (line 7)) (0.2.8)
[2023-03-08T17:31:01.513+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-logging>=2.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.2.1)
[2023-03-08T17:31:01.520+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc>=3.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (5.0.0)
[2023-03-08T17:31:01.541+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpx in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.23.3)
[2023-03-08T17:31:01.547+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-storage in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (8.0.0)
[2023-03-08T17:31:01.549+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-auth>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (4.1.5)
[2023-03-08T17:31:01.551+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-aiplatform<2.0.0,>=1.7.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.16.1)
[2023-03-08T17:31:01.554+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyOpenSSL in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (22.1.0)
[2023-03-08T17:31:01.557+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-pubsub>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.13.5)
[2023-03-08T17:31:01.560+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-bigquery>=1.2.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:31:01.563+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-kms>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.12.0)
[2023-03-08T17:31:01.565+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-videointelligence<2.0.0,>=1.7.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.16.3)
[2023-03-08T17:31:01.567+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-orchestration-***<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.4.1)
[2023-03-08T17:31:01.568+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dlp<2.0.0,>=0.11.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:31:01.570+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-build>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.9.0)
[2023-03-08T17:31:01.572+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-secret-manager<2.0.0,>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:31:01.574+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-python-client<2.0.0,>=1.6.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.12.11)
[2023-03-08T17:31:01.577+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-datatransfer>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.7.0)
[2023-03-08T17:31:01.579+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-workflows<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.7.1)
[2023-03-08T17:31:01.582+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-httplib2>=0.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.1.0)
[2023-03-08T17:31:01.584+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-gcp>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.2.2)
[2023-03-08T17:31:01.586+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc-metastore<2.0.0,>=1.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.6.0)
[2023-03-08T17:31:01.588+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-common-sql>=1.3.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.3.3)
[2023-03-08T17:31:01.589+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-translate<2.0.0,>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.7.2)
[2023-03-08T17:31:01.591+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataplex>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.1.0)
[2023-03-08T17:31:01.594+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-memcache>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.4.1)
[2023-03-08T17:31:01.596+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-container<3.0.0,>=2.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.11.1)
[2023-03-08T17:31:01.598+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-tasks>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.10.1)
[2023-03-08T17:31:01.605+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-bigquery>=6.1.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (6.2.0)
[2023-03-08T17:31:01.606+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-ads>=15.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (18.0.0)
[2023-03-08T17:31:01.609+0000] {process_utils.py:187} INFO - Requirement already satisfied: json-merge-patch>=0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.2)
[2023-03-08T17:31:01.611+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-redis>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.9.0)
[2023-03-08T17:31:01.613+0000] {process_utils.py:187} INFO - Requirement already satisfied: asgiref>=3.5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.6.0)
[2023-03-08T17:31:01.615+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-texttospeech<2.0.0,>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.0.3)
[2023-03-08T17:31:01.617+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-datacatalog>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.9.0)
[2023-03-08T17:31:01.619+0000] {process_utils.py:187} INFO - Requirement already satisfied: looker-sdk>=22.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (22.20.0)
[2023-03-08T17:31:01.621+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-spanner<2.0.0,>=1.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.19.3)
[2023-03-08T17:31:01.623+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-vision<2.0.0,>=0.35.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:31:01.624+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-automl>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.8.0)
[2023-03-08T17:31:01.634+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas-gbq in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.17.9)
[2023-03-08T17:31:01.640+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-compute<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.7.0)
[2023-03-08T17:31:01.646+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataform>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.2.0)
[2023-03-08T17:31:01.649+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigtable<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.7.3)
[2023-03-08T17:31:01.651+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-monitoring>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.11.0)
[2023-03-08T17:31:01.652+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-language<2.0.0,>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.3.2)
[2023-03-08T17:31:01.654+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-os-login>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.7.1)
[2023-03-08T17:31:01.655+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-speech<2.0.0,>=0.36.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.3.4)
[2023-03-08T17:31:06.783+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-wtf>=0.15 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.1.1)
[2023-03-08T17:31:06.785+0000] {process_utils.py:187} INFO - Requirement already satisfied: blinker in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.5)
[2023-03-08T17:31:06.787+0000] {process_utils.py:187} INFO - Requirement already satisfied: rich>=12.4.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (13.1.0)
[2023-03-08T17:31:06.790+0000] {process_utils.py:187} INFO - Requirement already satisfied: jsonschema>=3.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.17.3)
[2023-03-08T17:31:06.792+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-ftp in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.3.0)
[2023-03-08T17:31:06.793+0000] {process_utils.py:187} INFO - Requirement already satisfied: tabulate>=0.7.5 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:31:06.795+0000] {process_utils.py:187} INFO - Requirement already satisfied: attrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (22.2.0)
[2023-03-08T17:31:06.797+0000] {process_utils.py:187} INFO - Requirement already satisfied: pluggy>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.0.0)
[2023-03-08T17:31:06.798+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-login>=0.6.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.6.2)
[2023-03-08T17:31:06.801+0000] {process_utils.py:187} INFO - Requirement already satisfied: tenacity>=6.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (8.1.0)
[2023-03-08T17:31:06.803+0000] {process_utils.py:187} INFO - Requirement already satisfied: termcolor>=1.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.2.0)
[2023-03-08T17:31:06.805+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdit-py-plugins>=0.3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.3.3)
[2023-03-08T17:31:06.806+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-daemon>=2.2.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.3.2)
[2023-03-08T17:31:06.808+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-appbuilder==4.1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.1.4)
[2023-03-08T17:31:06.810+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-nvd3>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.15.0)
[2023-03-08T17:31:06.812+0000] {process_utils.py:187} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:31:06.814+0000] {process_utils.py:187} INFO - Requirement already satisfied: pygments>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.14.0)
[2023-03-08T17:31:06.815+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask<2.3,>=2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.2.2)
[2023-03-08T17:31:06.817+0000] {process_utils.py:187} INFO - Requirement already satisfied: gunicorn>=20.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (20.1.0)
[2023-03-08T17:31:06.819+0000] {process_utils.py:187} INFO - Requirement already satisfied: linkify-it-py>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:31:06.821+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown>=3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.4.1)
[2023-03-08T17:31:06.824+0000] {process_utils.py:187} INFO - Requirement already satisfied: setproctitle>=1.1.8 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.3.2)
[2023-03-08T17:31:06.825+0000] {process_utils.py:187} INFO - Requirement already satisfied: cattrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (22.2.0)
[2023-03-08T17:31:06.827+0000] {process_utils.py:187} INFO - Requirement already satisfied: psutil>=4.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (5.9.4)
[2023-03-08T17:31:06.828+0000] {process_utils.py:187} INFO - Requirement already satisfied: lockfile>=0.12.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.12.2)
[2023-03-08T17:31:06.830+0000] {process_utils.py:187} INFO - Requirement already satisfied: unicodecsv>=0.14.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.14.1)
[2023-03-08T17:31:06.831+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyjwt>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.6.0)
[2023-03-08T17:31:06.838+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-http in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.1.1)
[2023-03-08T17:31:06.839+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorlog<5.0,>=4.0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.8.0)
[2023-03-08T17:31:06.841+0000] {process_utils.py:187} INFO - Requirement already satisfied: croniter>=0.3.17 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.3.8)
[2023-03-08T17:31:06.843+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy<2.0,>=1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.4.46)
[2023-03-08T17:31:06.847+0000] {process_utils.py:187} INFO - Requirement already satisfied: cron-descriptor>=1.2.24 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.2.32)
[2023-03-08T17:31:06.849+0000] {process_utils.py:187} INFO - Requirement already satisfied: pathspec~=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:31:06.850+0000] {process_utils.py:187} INFO - Requirement already satisfied: argcomplete>=1.10 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:31:06.851+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-caching>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.0.2)
[2023-03-08T17:31:06.854+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-session>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.4.0)
[2023-03-08T17:31:06.856+0000] {process_utils.py:187} INFO - Requirement already satisfied: typing-extensions>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.4.0)
[2023-03-08T17:31:06.857+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown-it-py>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.1.0)
[2023-03-08T17:31:06.861+0000] {process_utils.py:187} INFO - Requirement already satisfied: configupdater>=3.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.1.1)
[2023-03-08T17:31:06.864+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-jsonfield>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.0.1.post0)
[2023-03-08T17:31:06.869+0000] {process_utils.py:187} INFO - Requirement already satisfied: cached-property>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.5.2)
[2023-03-08T17:31:06.871+0000] {process_utils.py:187} INFO - Requirement already satisfied: cryptography>=0.9.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (38.0.4)
[2023-03-08T17:31:06.873+0000] {process_utils.py:187} INFO - Requirement already satisfied: alembic<2.0,>=1.6.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.9.2)
[2023-03-08T17:31:06.874+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-sqlite in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.3.1)
[2023-03-08T17:31:06.876+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-slugify>=5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (7.0.0)
[2023-03-08T17:31:06.878+0000] {process_utils.py:187} INFO - Requirement already satisfied: connexion[flask]>=2.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.14.1)
[2023-03-08T17:31:06.880+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-metadata>=1.7 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.13.0)
[2023-03-08T17:31:06.882+0000] {process_utils.py:187} INFO - Requirement already satisfied: dill>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.3.1.1)
[2023-03-08T17:31:06.885+0000] {process_utils.py:187} INFO - Requirement already satisfied: lazy-object-proxy in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.9.0)
[2023-03-08T17:31:06.889+0000] {process_utils.py:187} INFO - Requirement already satisfied: deprecated>=1.2.13 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.2.13)
[2023-03-08T17:31:06.893+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-oneofschema>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.0.1)
[2023-03-08T17:31:06.895+0000] {process_utils.py:187} INFO - Requirement already satisfied: jinja2>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.1.2)
[2023-03-08T17:31:06.897+0000] {process_utils.py:187} INFO - Requirement already satisfied: pendulum>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:31:06.899+0000] {process_utils.py:187} INFO - Requirement already satisfied: werkzeug>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.2.2)
[2023-03-08T17:31:06.901+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-imap in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.1.1)
[2023-03-08T17:31:06.912+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-resources>=5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (5.10.2)
[2023-03-08T17:31:06.913+0000] {process_utils.py:187} INFO - Requirement already satisfied: markupsafe>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:31:06.915+0000] {process_utils.py:187} INFO - Requirement already satisfied: graphviz>=0.12 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.20.1)
[2023-03-08T17:31:06.923+0000] {process_utils.py:187} INFO - Requirement already satisfied: prison<1.0.0,>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.2.1)
[2023-03-08T17:31:06.925+0000] {process_utils.py:187} INFO - Requirement already satisfied: WTForms<4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.0.1)
[2023-03-08T17:31:06.926+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-utils<1,>=0.32.21 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.39.0)
[2023-03-08T17:31:06.928+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-JWT-Extended<5.0.0,>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (4.4.4)
[2023-03-08T17:31:06.930+0000] {process_utils.py:187} INFO - Requirement already satisfied: apispec[yaml]<4,>=3.3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.3.2)
[2023-03-08T17:31:06.931+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-SQLAlchemy<3,>=2.4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.5.1)
[2023-03-08T17:31:06.933+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-enum<2,>=1.5.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.5.1)
[2023-03-08T17:31:06.935+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-Babel<3,>=1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:31:06.937+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow<4,>=3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.19.0)
[2023-03-08T17:31:06.938+0000] {process_utils.py:187} INFO - Requirement already satisfied: click<9,>=8 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (8.1.3)
[2023-03-08T17:31:06.939+0000] {process_utils.py:187} INFO - Requirement already satisfied: email-validator<2,>=1.0.5 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.3.0)
[2023-03-08T17:31:06.941+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorama<1,>=0.3.9 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.4.6)
[2023-03-08T17:31:06.945+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-sqlalchemy<0.27.0,>=0.22.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.26.1)
[2023-03-08T17:31:07.185+0000] {process_utils.py:187} INFO - Collecting tqdm
[2023-03-08T17:31:07.192+0000] {process_utils.py:187} INFO -   Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
[2023-03-08T17:31:07.225+0000] {process_utils.py:187} INFO - Requirement already satisfied: asynctest==0.13.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (0.13.0)
[2023-03-08T17:31:07.227+0000] {process_utils.py:187} INFO - Requirement already satisfied: charset-normalizer<3.0,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (2.1.1)
[2023-03-08T17:31:07.229+0000] {process_utils.py:187} INFO - Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (4.0.2)
[2023-03-08T17:31:07.232+0000] {process_utils.py:187} INFO - Requirement already satisfied: multidict<7.0,>=4.5 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (6.0.4)
[2023-03-08T17:31:07.234+0000] {process_utils.py:187} INFO - Requirement already satisfied: yarl<2.0,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (1.8.2)
[2023-03-08T17:31:07.236+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiosignal>=1.1.2 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (1.3.1)
[2023-03-08T17:31:07.237+0000] {process_utils.py:187} INFO - Requirement already satisfied: frozenlist>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (1.3.3)
[2023-03-08T17:31:07.255+0000] {process_utils.py:187} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.7/site-packages (from alembic<2.0,>=1.6.3->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.2.4)
[2023-03-08T17:31:07.282+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlparse>=0.4.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-common-sql>=1.3.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.4.3)
[2023-03-08T17:31:07.398+0000] {process_utils.py:187} INFO - Requirement already satisfied: exceptiongroup in /home/***/.local/lib/python3.7/site-packages (from cattrs>=22.1.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.1.0)
[2023-03-08T17:31:07.492+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyYAML<7,>=5.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (6.0)
[2023-03-08T17:31:07.495+0000] {process_utils.py:187} INFO - Requirement already satisfied: inflection<0.6,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.5.1)
[2023-03-08T17:31:07.498+0000] {process_utils.py:187} INFO - Requirement already satisfied: clickclick<21,>=1.2 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (20.10.2)
[2023-03-08T17:31:07.558+0000] {process_utils.py:187} INFO - Requirement already satisfied: cffi>=1.12 in /home/***/.local/lib/python3.7/site-packages (from cryptography>=0.9.3->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.15.1)
[2023-03-08T17:31:07.605+0000] {process_utils.py:187} INFO - Requirement already satisfied: wrapt<2,>=1.10 in /home/***/.local/lib/python3.7/site-packages (from deprecated>=1.2.13->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.14.1)
[2023-03-08T17:31:07.650+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachelib<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from flask-caching>=1.5.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:31:07.716+0000] {process_utils.py:187} INFO - Requirement already satisfied: chardet<4.1,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (4.0.0)
[2023-03-08T17:31:07.719+0000] {process_utils.py:187} INFO - Requirement already satisfied: future<0.19.0,>=0.17.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.18.3)
[2023-03-08T17:31:07.721+0000] {process_utils.py:187} INFO - Requirement already satisfied: backoff<3.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.10.0)
[2023-03-08T17:31:07.762+0000] {process_utils.py:187} INFO - Requirement already satisfied: googleapis-common-protos<2.0.0,>=1.5.8 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.56.4)
[2023-03-08T17:31:07.764+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-status<2.0.0,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.48.2)
[2023-03-08T17:31:07.767+0000] {process_utils.py:187} INFO - Requirement already satisfied: setuptools>=40.3.0 in /tmp/venvsm2wgtk3/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (67.2.0)
[2023-03-08T17:31:07.844+0000] {process_utils.py:187} INFO - Requirement already satisfied: httplib2<1dev,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.20.4)
[2023-03-08T17:31:07.847+0000] {process_utils.py:187} INFO - Requirement already satisfied: uritemplate<4dev,>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.0.1)
[2023-03-08T17:31:07.881+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-oauthlib>=0.7.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth-oauthlib->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (1.3.1)
[2023-03-08T17:31:08.036+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-resource-manager<3.0.0dev,>=1.3.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-aiplatform<2.0.0,>=1.7.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.6.0)
[2023-03-08T17:31:08.103+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpc-google-iam-v1<0.13dev,>=0.12.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigtable<2.0.0,>=1.0.0->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.12.4)
[2023-03-08T17:31:08.457+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-appengine-logging<2.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.1.3)
[2023-03-08T17:31:08.467+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-audit-log<1.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.2.4)
[2023-03-08T17:31:09.050+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from google-resumable-media>=2.3.2->google-cloud-storage->-r /tmp/venvsm2wgtk3/requirements.txt (line 5)) (1.5.0)
[2023-03-08T17:31:09.254+0000] {process_utils.py:187} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.7/site-packages (from importlib-metadata>=1.7->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (3.11.0)
[2023-03-08T17:31:09.451+0000] {process_utils.py:187} INFO - Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.3.10)
[2023-03-08T17:31:09.453+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.19.3)
[2023-03-08T17:31:09.546+0000] {process_utils.py:187} INFO - Requirement already satisfied: uc-micro-py in /home/***/.local/lib/python3.7/site-packages (from linkify-it-py>=2.0.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.0.1)
[2023-03-08T17:31:09.721+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdurl~=0.1 in /home/***/.local/lib/python3.7/site-packages (from markdown-it-py>=2.1.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.1.2)
[2023-03-08T17:31:09.882+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/***/.local/lib/python3.7/site-packages (from packaging<22.0dev,>=14.3->google-cloud-bigquery->-r /tmp/venvsm2wgtk3/requirements.txt (line 6)) (3.0.9)
[2023-03-08T17:31:09.942+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytzdata>=2020.1 in /home/***/.local/lib/python3.7/site-packages (from pendulum>=2.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2020.1)
[2023-03-08T17:31:10.099+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/***/.local/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth->-r /tmp/venvsm2wgtk3/requirements.txt (line 7)) (0.4.8)
[2023-03-08T17:31:10.257+0000] {process_utils.py:187} INFO - Requirement already satisfied: docutils in /home/***/.local/lib/python3.7/site-packages (from python-daemon>=2.2.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.19)
[2023-03-08T17:31:10.366+0000] {process_utils.py:187} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.7/site-packages (from python-slugify>=5.0->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (1.3)
[2023-03-08T17:31:10.453+0000] {process_utils.py:187} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.7/site-packages (from requests->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (3.4)
[2023-03-08T17:31:10.522+0000] {process_utils.py:187} INFO - Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from rich>=12.4.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.9.1)
[2023-03-08T17:31:10.809+0000] {process_utils.py:187} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy<2.0,>=1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.0.1)
[2023-03-08T17:31:10.875+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-storage<3.0.0dev,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (2.14.1)
[2023-03-08T17:31:10.878+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyarrow>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (9.0.0)
[2023-03-08T17:31:11.147+0000] {process_utils.py:187} INFO - Collecting outcome
[2023-03-08T17:31:11.153+0000] {process_utils.py:187} INFO -   Using cached outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
[2023-03-08T17:31:11.157+0000] {process_utils.py:187} INFO - Requirement already satisfied: sniffio in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venvsm2wgtk3/requirements.txt (line 1)) (1.3.0)
[2023-03-08T17:31:11.158+0000] {process_utils.py:187} INFO - Requirement already satisfied: sortedcontainers in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venvsm2wgtk3/requirements.txt (line 1)) (2.4.0)
[2023-03-08T17:31:11.200+0000] {process_utils.py:187} INFO - Collecting async-generator>=1.9
[2023-03-08T17:31:11.205+0000] {process_utils.py:187} INFO -   Using cached async_generator-1.10-py3-none-any.whl (18 kB)
[2023-03-08T17:31:11.304+0000] {process_utils.py:187} INFO - Collecting wsproto>=0.14
[2023-03-08T17:31:11.308+0000] {process_utils.py:187} INFO -   Using cached wsproto-1.2.0-py3-none-any.whl (24 kB)
[2023-03-08T17:31:11.503+0000] {process_utils.py:187} INFO - Collecting PySocks!=1.5.7,<2.0,>=1.5.6
[2023-03-08T17:31:11.507+0000] {process_utils.py:187} INFO -   Using cached PySocks-1.7.1-py3-none-any.whl (16 kB)
[2023-03-08T17:31:11.626+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-toolbelt in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-http->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (0.10.1)
[2023-03-08T17:31:11.807+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiofiles<23.0.0,>=0.6.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-storage->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (22.1.0)
[2023-03-08T17:31:11.874+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpcore<0.17.0,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.16.3)
[2023-03-08T17:31:11.876+0000] {process_utils.py:187} INFO - Requirement already satisfied: rfc3986[idna2008]<2,>=1.3 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:31:12.013+0000] {process_utils.py:187} INFO - Requirement already satisfied: db-dtypes<2.0.0,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.0.5)
[2023-03-08T17:31:12.021+0000] {process_utils.py:187} INFO - Requirement already satisfied: pydata-google-auth in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:31:12.614+0000] {process_utils.py:187} INFO - Requirement already satisfied: pycparser in /home/***/.local/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=0.9.3->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.21)
[2023-03-08T17:31:12.990+0000] {process_utils.py:187} INFO - Requirement already satisfied: dnspython>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from email-validator<2,>=1.0.5->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.3.0)
[2023-03-08T17:31:13.123+0000] {process_utils.py:187} INFO - Requirement already satisfied: Babel>=2.3 in /home/***/.local/lib/python3.7/site-packages (from Flask-Babel<3,>=1->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvsm2wgtk3/requirements.txt (line 9)) (2.11.0)
[2023-03-08T17:31:13.375+0000] {process_utils.py:187} INFO - WARNING: google-api-core 2.8.2 does not provide the extra 'grpcgcp'
[2023-03-08T17:31:13.958+0000] {process_utils.py:187} INFO - Requirement already satisfied: anyio<5.0,>=3.0 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (3.6.2)
[2023-03-08T17:31:13.961+0000] {process_utils.py:187} INFO - Requirement already satisfied: h11<0.15,>=0.13 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venvsm2wgtk3/requirements.txt (line 8)) (0.14.0)
[2023-03-08T17:31:15.117+0000] {process_utils.py:187} INFO - Requirement already satisfied: oauthlib>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib->gcsfs->-r /tmp/venvsm2wgtk3/requirements.txt (line 3)) (3.2.2)
[2023-03-08T17:31:18.416+0000] {process_utils.py:187} INFO - Installing collected packages: tqdm, PySocks, outcome, fsspec, async-generator, wsproto, trio, trio-websocket, openai, selenium, gcsfs
[2023-03-08T17:31:18.513+0000] {process_utils.py:187} INFO -   Attempting uninstall: fsspec
[2023-03-08T17:31:18.518+0000] {process_utils.py:187} INFO -     Found existing installation: fsspec 2022.11.0
[2023-03-08T17:31:18.521+0000] {process_utils.py:187} INFO -     Not uninstalling fsspec at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venvsm2wgtk3
[2023-03-08T17:31:18.524+0000] {process_utils.py:187} INFO -     Can't uninstall 'fsspec'. No files were found to uninstall.
[2023-03-08T17:31:20.404+0000] {process_utils.py:187} INFO - Successfully installed PySocks-1.7.1 async-generator-1.10 fsspec-2023.1.0 gcsfs-2023.1.0 openai-0.27.0 outcome-1.2.0 selenium-4.8.2 tqdm-4.65.0 trio-0.22.0 trio-websocket-0.9.2 wsproto-1.2.0
[2023-03-08T17:31:20.867+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvsm2wgtk3/bin/python /tmp/venvsm2wgtk3/script.py /tmp/venvsm2wgtk3/script.in /tmp/venvsm2wgtk3/script.out /tmp/venvsm2wgtk3/string_args.txt
[2023-03-08T17:31:20.892+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:31:24.784+0000] {process_utils.py:187} INFO - /home/***/.local/lib/python3.7/site-packages/***/models/base.py:49 MovedIn20Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
[2023-03-08T17:31:28.698+0000] {process_utils.py:187} INFO - Traceback (most recent call last):
[2023-03-08T17:31:28.700+0000] {process_utils.py:187} INFO -   File "/tmp/venvsm2wgtk3/script.py", line 54, in <module>
[2023-03-08T17:31:28.704+0000] {process_utils.py:187} INFO -     res = chat_gpt_prediction(*arg_dict["args"], **arg_dict["kwargs"])
[2023-03-08T17:31:28.707+0000] {process_utils.py:187} INFO -   File "/tmp/venvsm2wgtk3/script.py", line 35, in chat_gpt_prediction
[2023-03-08T17:31:28.709+0000] {process_utils.py:187} INFO -     df = pd.read_csv(os.path.join(data_dir,'bitcoin_pricing.csv'), header=0)
[2023-03-08T17:31:28.713+0000] {process_utils.py:187} INFO - NameError: name 'pd' is not defined
[2023-03-08T17:31:29.324+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 356, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 553, in execute_callable
    return self._execute_python_callable_in_subprocess(python_path, tmp_path)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 418, in _execute_python_callable_in_subprocess
    os.fspath(string_args_path),
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 168, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 191, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venvsm2wgtk3/bin/python', '/tmp/venvsm2wgtk3/script.py', '/tmp/venvsm2wgtk3/script.in', '/tmp/venvsm2wgtk3/script.out', '/tmp/venvsm2wgtk3/string_args.txt']' returned non-zero exit status 1.
[2023-03-08T17:31:29.358+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=bitcoin_second_webscrape, task_id=chatgpt_prediction, execution_date=20230308T170028, start_date=20230308T173055, end_date=20230308T173129
[2023-03-08T17:31:29.393+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 218 for task chatgpt_prediction (Command '['/tmp/venvsm2wgtk3/bin/python', '/tmp/venvsm2wgtk3/script.py', '/tmp/venvsm2wgtk3/script.in', '/tmp/venvsm2wgtk3/script.out', '/tmp/venvsm2wgtk3/string_args.txt']' returned non-zero exit status 1.; 2009)
[2023-03-08T17:31:29.435+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-08T17:31:29.464+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
